frontend/views/components/banners/BannerSimple.vue

```diff
@@ -16,13 +16,13 @@ export default {
   props: {
     severity: {
       type: String,
-      default: 'info'
+      default: 'info',
+      validator: function (value) {
```

Comment:
This change is not relevant to the issue but thought it was obvious this prop validation function was misplaced, so boyscouted it. But pls let me know if it needs to be put back.

Reply:
Your change looks correct to me, nice catch! 👍